src/aap_eda/services/activation/engine/podman.py

```diff
@@ -334,9 +334,7 @@ def _login(self, request: ContainerRequest) -> None:
                 registry=registry,
             )
 
-            LOGGER.debug(
-                f"{credential.username} login succeeded to {registry}"
-            )
+            LOGGER.info(f"{credential.username} login succeeded to {registry}")
```

Comment:
Maybe quote the username, so it's easier to spot?

Reply:
updated

Reply:
You are right. Can you add a TODO to remove it? It might be worth to check for dead code, I feel there is more.

Reply:
TODO added